### ISSUE TITLE
Design Suggestion: Condensing Editor Bar

### ISSUE THREAD

**Issue by davidsword** [WordPress/gutenberg#3160]
The original design had more and lengthier tools in the editor bar:

![screen shot 2017-10-25 at 2 47 23 pm](https://user-images.githubusercontent.com/1996370/32025062-a39078fe-b993-11e7-99ef-8de87994fc7e.png)

---

But in 1.5.1, there's less tools, and their length shorter. It feels like tools are scattered here-and-there (namely the info icon thats too good for the editor bar or the sidebar). 

This is all especially noticeable on monitors bigger than laptops (keep in mind these mockups are around 1000px wide).

![1](https://user-images.githubusercontent.com/1996370/32025102-cdd80ee2-b993-11e7-848b-a3339cbbf684.jpg)

---

What if the tools got condensed to the top right, length equal to the Gutenberg sidebar - giving us a three column layout:

* **Left:** Wordpress sidebar
* **Center:** Editor
* **Right:** Editor sidebar

Something like this

![2](https://user-images.githubusercontent.com/1996370/32025140-ed961666-b993-11e7-97a9-c050b24522f4.jpg)

---

When collapsing the Gutenberg sidebar, a bit of header padding remains and the essential tools remain.

![3](https://user-images.githubusercontent.com/1996370/32025177-125e92e8-b994-11e7-963f-edb598f32f2d.jpg)

---

Just throwing it out there. 

** this includes #3152  solution for repeat of Publish settings

**Comment by kevinwhoffman** [comment]
I appreciate the exploration. I also was left wanting to clean up the controls after 1.5 released. Unfortunately I think this takes us farther down the path of ambiguous icons. Other than the undo/redo arrows, the other icons are not obvious in their meaning.

I do like that this brings some consistency to the controls appearance, but I think we can do better in terms of clarifying purpose.

**Comment by jasmussen** [comment]
Thanks so much for the exploration!

Concerns like those posted here are part of why we redesigned the editor bar in 1.5, but another reason was accessibility, to group items in a better way, and to accomodate a contextual toolbar. 

As much as I like icon only buttons for some things, it definitely is harder to read at a glance, and so removing the label should only happen after we've used and tested the tool and know for sure it's "safe" to remove the label.

Combined with a desire to keep the UI consistent with mobile, I'm going to close this issue for now, but we can always revisit if this comes up again! 

By the way I agree with you that the Info icon deserves a little love. Perhaps we can move that one to the editor bar, when you have no block selected? Definitely worth exploring separately. Thanks for your input!
